code/datums/keybindings/_keybinding.dm

```diff
@@ -0,0 +1,59 @@
+/datum/keybinding
+	/// The user-facing name.
+	var/name
+	/// The UI category to belong to.
+	var/category = KB_CATEGORY_UNSORTED
+	/// The default key(s) assigned to the keybind.
+	var/list/keys
+	/// Whether the keybind should add the client to the SSinput processing loop. Use VERY sparingly.
+	var/key_loop = FALSE
```

Comment:
Why have this when you already have `should_start_looping` for this? Seems a bit double. Just override that one to return `TRUE` instead